### DISCUSSION TITLE
Is there any way bulk import JSON data

### DISCUSSION THREAD

**Discussion by indahud** [supabase/supabase#705] [Questions] [answered]
I am looking to add 100 records once. The data is array of Objects.

**Comment by inian** [comment, marked as answer]
Hi @indahud, if you are planning to add 100 records, you can use any of our [client libraries](https://supabase.io/docs/client/supabase-client). [Here](https://supabase.io/docs/client/insert) is an example to add an array of JS objects using our JS client. 

If you are planning to add thousands or millions of records, it might be faster to connect to the database via `psql` and insert records directly.

**Reply by kiwicopple** [reply]
This could also be helpful if you go down the `psql` route: https://dev.to/kiwicopple/loading-json-into-postgres-2l28

**Reply by indahud** [reply]
Thank you.
